src/test/kotlin/org/ionproject/integration/application/job/chunkbased/FormatVerifierStepTest.kt

```diff
@@ -232,24 +195,10 @@ internal class FormatVerifierStepTestInvalidFormat {
     }
 
     private val utils = SpringBatchTestUtils()
-    private lateinit var testSmtp: GreenMail
-
-    @BeforeEach
-    fun testSmtpInit() {
-        Security.setProperty("ssl.SocketFactory.provider", DummySSLSocketFactory::class.java.name)
-        testSmtp = GreenMail(ServerSetupTest.SMTP)
-        testSmtp.start()
-    }
-
-    @AfterEach
-    fun stopMailServer() {
-        testSmtp.stop()
-    }
 
-    @Test
+    // @Test
```

Comment:
I think this was forgotten

Reply:
This was failing due to the email sending was forgotten there.
The notifications (log, email, etc) should come from the job Listeners implemented in `NotificationListener.kt`, so I have removed the email sending from `AlertOnFailureWriter.kt`.  The test aims to test the format of the file and is now running successfully.